external-api/api-tests/v1/party-db.api-test.ts

```diff
@@ -37,6 +45,13 @@ describe('/party-db', () => {
 
       expect(status).toEqual(HttpStatusCode.Ok);
     });
+
+    it(`calls the correct url`, async () => {
+      await get(`/party-db/${VALID}`);
+
+      expect(axiosMock.history.get).toHaveLength(1);
```

Comment:
Instead of `history` can we spy on the function and assert the calling function and arguments please to keep it consistent with testing methodoloy please.